### ISSUE TITLE
aws-eks/create-cluster is generating error at the install-eksctl step

### ISSUE THREAD

**Issue by SudKul** [CircleCI-Public/aws-eks-orb#47]
### Orb version
```bash
  orbs:
  aws-eks: circleci/aws-eks@0.2.3
  kubernetes: circleci/kubernetes@0.4.0
```

### What happened

While creating a new AWS EKS cluster using this [example config](https://circleci.com/developer/orbs/orb/circleci/aws-eks#usage-create-eks-cluster) file, it generates an error at the `install-eksctl` step. Below is the output from the CircleCI failed build. 
```bash
#!/bin/bash -eo pipefail
if which eksctl > /dev/null; then
  echo "eksctl is already installed"
  exit 0
fi

mkdir -p eksctl_download
curl --silent --location --retry 5 "https://github.com/weaveworks/eksctl/releases/download/latest_release/eksctl_$(uname -s)_amd64.tar.gz" \
  | tar xz -C eksctl_download
chmod +x eksctl_download/eksctl
SUDO=""
if [ $(id -u) -ne 0 ] && which sudo > /dev/null ; then
  SUDO="sudo"
fi
$SUDO mv eksctl_download/eksctl /usr/local/bin/
rmdir eksctl_download


gzip: stdin: not in gzip format
tar: Child returned status 1
tar: Error is not recoverable: exiting now

Exited with code exit status 2
CircleCI received exit code 2
```
See the snapshot of the error below.
You can see the files in this [commit](https://github.com/SudKul/nd9991-Capstonoe/commit/027cb65cbb75e8f6d1d95ddd726aa354326a0280). 

<img width="785" alt="Screenshot 2021-11-10 at 1 45 51 AM" src="https://user-images.githubusercontent.com/48475411/140998705-4c3fd8f4-5a39-485e-a2af-b33ec85c4b45.png">

### Expected behavior

The example should install `eksctl` and create a cluster successfully.

**Comment by superafroman** [comment]
Hi @SudKul, not sure what the benefit is of opening a new issue when #42 already exists (you've marked it as a duplicate) and is being resolved in #46?

**Comment by SudKul** [comment]
My apologies for not searching through the existing list of issues.
